internal_review/housing_security/borough/units_rentstable.csv

```diff
@@ -0,0 +1,6 @@
+borough,units_rentstable,units_rentstable_moe,units_rentstable_cv,units_rentstable_pct,units_rentstable_pct_moe,units_occurental,units_occurental_moe,units_occurental_cv
```

Comment:
I think we want to round all the decimals (x.xx) but double check with @AmandaDoyle